src/fields/slider.js

```diff
@@ -12,8 +12,11 @@ function Slider (name, initialValue, config, parentField) {
 
   Field.call(this, name, initialValue, parentField, config);
 
-  var min = config.min === undefined ? Math.min(initialValue, 0) : config.min;
-  var max = config.max === undefined ? Math.max(initialValue, 1) : config.max;
+  var isValueBetween0and1 = 0 <= initialValue && initialValue <= 1;
+  var defaultMin = isValueBetween0and1 ? 0 : Math.min(initialValue * 2, 0);
```

Comment:
Took me a moment, but this handles negative numbers just fine, right? Nice solution.

Reply:
Yup, it was not easy 🙂 
Basically this doubles the negative min also if the provided number is negative. So it goes from `-double` to `0`